engines/dradis-api/lib/dradis/ce/api/api_token_strategy.rb

```diff
@@ -0,0 +1,50 @@
+module Dradis::CE::API
+  class ApiTokenStrategy < ::Warden::Strategies::Base
+    attr_reader :email, :password
+
+    # This strategy should be applied when we are requesting /api/
+    def valid?
+      !!(request.path_info =~ /\A\/api\//)
```

Comment:
on a separate note, our second API strategy is missing the `path_info` check we have here, I think we should add it as well.